scripts/contiv-vpp-bug-report.sh

```diff
@@ -133,9 +152,30 @@ for POD in $PODS; do
     # get_vpp_data "api trace custom-dump /tmp/trace.api" api-trace-dump
     echo
     popd >/dev/null
-
 done
 
+KSR_POD="$(master_kubectl get po -n kube-system -l k8s-app=contiv-ksr -o "'go-template={{range .items}}{{printf \"%s,%s\" (index .metadata).name (index .spec).nodeName}}{{end}}'")"
```

Comment:
Can you please break up lines over 120 chars?  Super long ones are hard to read.